pkg/skaffold/errors/errors.go

```diff
@@ -71,6 +71,15 @@ func ShowAIError(err error) error {
 			}
 		}
 	}
+
+	for _, v := range knownDeployProblems {
```

Comment:
you could drop repeating the loop by appending `knownBuildProblems` and `knownDeployProblems`
https://tour.golang.org/moretypes/15